pkg/podvolume/backupper.go

```diff
@@ -107,7 +110,13 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 		return nil, nil
 	}
 
-	repo, err := b.repoEnsurer.EnsureRepo(b.ctx, backup.Namespace, pod.Namespace, backup.Spec.StorageLocation)
+	repositoryType := getRepositoryType(b.uploaderType)
+	if repositoryType == "" {
+		err := errors.New("invalid repository type")
```

Comment:
In this case there's probably a wrong uploadertype, so would it be helpful to put the uploader type as part of the error?

Reply:
Done, added the uploader type